src/components/Workspace.jsx

```diff
@@ -133,9 +138,88 @@ export default class Workspace extends React.Component {
     );
   }
 
+  _renderHiddenLanguages() {
+    const {
+      currentProject,
+      hiddenLanguages,
+      onComponentToggle,
+    } = this.props;
+    return (
+      <React.Fragment>
+        {hiddenLanguages.map(({language}) =>
+          (
+            <CollapsedComponent
+              component={`editor.${language}`}
+              key={language}
+              projectKey={currentProject.projectKey}
+              text={t(`languages.${language}`)}
+              onComponentUnhide={onComponentToggle}
+            />
+          ))}
+      </React.Fragment>
+    );
+  }
+
+  _renderHiddenLeftColumnComponents() {
+    const {
+      hiddenLanguages,
+    } = this.props;
+
+    if (
+      hiddenLanguages.length === 0 ||
+      hiddenLanguages.length === LANGUAGES.length
```

Comment:
I think it would make semantic sense to delegate this to `!_shouldRenderLeftColumn()`

Reply:
went with `_shouldRenderHiddenLeftColumnComponents` to match